guide/english/cplusplus/vector/index.md

```diff
@@ -231,5 +232,26 @@ int main(){
  return 0;
 }
 ```
+An alternative way to do this.
+
+``` cpp
+#include <iostream>
+#include <vector>
+#include <algorithm>
+using namespace std;
+
+int main(){
+ 
+ vector<int> v{ 10, 5, 82, 69, 64, 70, 3, 42, 28, 0 };
+ sort(v.rbegin(), v.rend());
```

Comment:
Please change this so the vector is sorted in ascending order, or change the output below.